modules/controlfreec/1.1/controlfreec.smk

```diff
@@ -34,20 +35,6 @@ localrules:
 
 ##### RULES #####
 
-
-# Symlinks the input files into the module results directory (under '00-inputs/')
-rule _controlfreec_input_bam:
-    input:
-        bam = CFG["inputs"]["sample_bam"],
-        bai = CFG["inputs"]["sample_bai"]
-    output:
-        bam = CFG["dirs"]["inputs"] + "bam/{seq_type}--{genome_build}/{sample_id}.bam",
-        bai = CFG["dirs"]["inputs"] + "bam/{seq_type}--{genome_build}/{sample_id}.bai"
-    run:
-        op.relative_symlink(input.bam, output.bam)
-        op.relative_symlink(input.bai, output.bai)
-        
-
 # generate references
```

Comment:
Does it need to be in reference_files?

Reply:
Yes please, also I don't think that you can assume that the user has `references/{genome_build}` configured as the location for their reference files. For modules I've written where I need to get specific, small reference files, I put them in the module `00-inputs` directory. That can be a stopgap until the rest is incorporated in the reference workflow.

Reply:
Hm I see, so just move these references as a subdirectory under 00-input?

ex. CFG["dirs"]["inputs"] + "references/{genome_build}/[...]"

Reply:
Yep!